ecf/hurricane/plots/jevs_hurricane_global_det_tropcyc_plots.ecf

```diff
@@ -15,7 +15,7 @@
 ############################################################
 # Load modules
 ############################################################
-module purge
+module reset
 export HPC_OPT=/apps/ops/prod/libs
 module use /apps/ops/prod/libs/modulefiles/compiler/intel/19.1.3.304/
```

Comment:
Use ${intel_ver} in place of 19.1.3.304. ${intel_ver} is defined in `versions/run.ver`. You will need to move these 2 lines:
`export HOMEevs=/lfs/h2/emc/vpppg/save/$USER/EVS`
`source ${HOMEevs}/versions/run.ver`
above this line:
`module use /apps/ops/prod/libs/modulefiles/compiler/intel/19.1.3.304/`
in order to use ${intel_ver} to remove usage of explicit version numbers.